lighthouse-extension/gulpfile.js

```diff
@@ -178,6 +207,14 @@ gulp.task('watch', ['lint', 'browserify', 'html'], () => {
     'app/src/**/*.js',
     '../lighthouse-core/**/*.js'
   ], ['browserify']);
+
+  gulp.watch([
+    '../lighthouse-core/**/*.html'
```

Comment:
this and the partials wildcard should overlap, right? does the order of these compile tasks matter?